File: vlc-android/src/org/videolan/vlc/gui/tv/AboutActivity.java
package org.videolan.vlc.gui.tv;

import android.os.Bundle;
import android.support.v4.app.FragmentActivity;
import android.widget.TextView;

import org.videolan.vlc.R;
import org.videolan.vlc.gui.helpers.UiTools;

public class AboutActivity extends FragmentActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.about_main);
        UiTools.fillAboutView(getWindow().getDecorView().getRootView());
        TvUtil.INSTANCE.applyOverscanMargin(this);
        TimeUpdaterKt.registerTimeView(this, (TextView) findViewById(R.id.tv_time));
    }
}

File: vlc-android/src/org/videolan/vlc/gui/tv/AboutActivity.kt
package org.videolan.vlc.gui.tv

import android.os.Bundle
import androidx.fragment.app.FragmentActivity

import org.videolan.vlc.R
import org.videolan.vlc.gui.helpers.UiTools

class AboutActivity : androidx.fragment.app.FragmentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.about_main)
        UiTools.fillAboutView(window.decorView.rootView)
        TvUtil.applyOverscanMargin(this)
        this.registerTimeView(findViewById(R.id.tv_time))
    }
}
